packages/nextjs/src/types/index.ts

```diff
@@ -78,6 +78,23 @@ export interface FronteggProviderProps extends FronteggProviderOptions {
   appName?: string;
 }
 
+export type CustomLoginOptionsType = {
```

Comment:
I think you can make the type in a way that will not allow to provide the two of them. I would go with aff a field `methodology/strategy/type: 'paramKey' | 'subDomain'` and for each of them hold an interface of whats relevant for them. (I did something similar with socialLoginButtonLayout which if its stackmode gives you some option and if its even give you different interface..)

Reply:
another options is:

```typescript
type ParamKeyProps = {
  paramKey: string;
}
type SubDomainProps = {
  subDomainIndex: number;
}

type PageDirectoryProviderProps = {
  customLoginOptions?:  ParamKeyProps |  SubDomainProps;
}
```